### DISCUSSION TITLE
Mixed Layer Depth in SubAntarctic Zone

### DISCUSSION THREAD

**Discussion by justinsmall** [NCAR/omwg_dev#39] [Biases]
Here are some plots of MLD in winter, based on the 0.03kg/m3 threshold, for the z* and hybrid control, compared to Argo.
![Slide1](https://github.com/NCAR/omwg_dev/assets/22841083/6fbf3953-85ec-44b4-97b6-0eade2980665)

**Comment by justinsmall** [comment]
Please note that these MLD biases in MOM6 are improved (i.e. MLD less deep) when using the submeso parameterization. Gustavo showed this at the winter WG meeting. I can add my own figures later.

**Comment by gustavo-marques** [comment]
Below are winter mean mixed layer depths for cases [018](https://github.com/NCAR/omwg_dev/issues/25) and [046](https://github.com/NCAR/omwg_dev/issues/47):

![image](https://github.com/NCAR/omwg_dev/assets/11339137/63d2fa84-408a-4e39-8c39-899224311b66)
![image](https://github.com/NCAR/omwg_dev/assets/11339137/323ccb9e-35ef-4c99-83a1-55ee10a86557)

 and the difference between them, 046 - 018:
  
![image](https://github.com/NCAR/omwg_dev/assets/11339137/a9029789-d148-4aa0-b806-d8692b2978c3)

Limiting the layer thicknesses, as done in 046, improves (shallower) the Winter MLD in the Subantarctic Zone.

**Comment by gustavo-marques** [comment]
Here is another comparison of the winter mean mixed layer depth for cases [018](https://github.com/NCAR/omwg_dev/issues/25) and case Bodner23.001 (not cataloged). These two cases have many differences, but the most relevant change is `USE_BODNER23 = True` in the latter.

![image](https://github.com/NCAR/omwg_dev/assets/11339137/599ae32b-f61b-4c26-873c-1e869b930f6c)

### Bodner23.001 - 018

![image](https://github.com/NCAR/omwg_dev/assets/11339137/226b6be7-9b36-494c-9281-d8ee6a6b8cbd)

As @justinsmall pointed out, using the BODNER23 scheme improves the Winter MLD biases in the Subantarctic Zone. However, it also shifts the location of maximum convection in the Labrador Sea. Below is the DeBoyer 2004 Winter Mean MLD climatology for comparison:
![image](https://github.com/NCAR/omwg_dev/assets/11339137/f93f9762-13f5-4da8-8076-c0499421e195)

**Comment by justinsmall** [comment]
@gustavo-marques Gustavo, is limiting the layer thickness a recommended idea? I also wonder if limiting layer thickness and applying Bodner param. will be effective - or will it reduce MLD in SubAntarctic too much?
It is very odd that the Bodner param. shifts the Labrador Sea convection eastwards. This maybe some indirect effect of changing density and flow?

**Comment by gustavo-marques** [comment]
@justinsmall: These are all good questions. 

Some thickness limiting (H_max) is recommended to prevent layers from getting excessively thick. So far we have been using a constant (in the vertical) limiting of ~ 400. The new approach is to apply H_max(z) as follows:

`H_max = 2.5, 2.5, 2.5, 2.5, 4.65954471, 4.91196546, 5.19412109, 5.50946059, 5.86181952, 6.25546, 6.69511391, 7.18602944, 7.73402094, 8.34552182, 9.02764048, 9.78821859, 10.63589128, 11.58014822, 12.63139452, 13.80100975, 15.10140314, 16.54606232, 18.14959251, 19.92774221, 21.89741083, 24.07663286, 26.4845323, 29.14124041, 32.06776905, 35.28583156, 38.81760291, 42.68541118, 46.91135357, 51.51683185, 56.52200507, 61.94516131, 67.80201558, 74.1049473, 80.86219824, 88.07705989, 95.74708725, 103.86338297, 112.41000134, 121.36352345, 130.69285331, 140.35927768, 150.31682062, 160.51290687, 170.8893278, 181.38348107, 191.92983316, 202.46153511, 212.91210815, 223.21710955, 233.31569094, 243.15197092, 252.67615996, 261.84539616, 270.62427305, 278.98506237, 286.90765387, 294.37924899, 301.39385482, 307.95162933, 314.05812899, 319.72350586, 324.96169559, 329.78962955, 334.22649664, 338.29307219, 342.01112468, 345.40290465, 348.49071585, 351.29656501, 353.84188435 ;`



However, we must still understand how thickness limiting relates to spurious mixing. We are actively exploring different options and evaluating their limitations. 

We should test the layer limiting + Bodner to verify what happens with the SubAntarctic MLDs. Yes, it's odd that the Bodner scheme shifts the convection in the Lab Sea, and this could be an indirect effect, as you pointed out.
